extensions/gopass/src/gopass.ts

```diff
@@ -47,10 +74,34 @@ async function otp(entry: string): Promise<string> {
   return gopass(["otp", "--password", entry]);
 }
 
+async function sync(): Promise<void> {
+  await gopass(["sync"]);
+}
+
 async function clip(entry: string): Promise<void> {
   await gopass(["show", "--clip", entry]);
 }
 
+async function remove(entry: string, force = false): Promise<void> {
+  await gopass(["remove", `--force=${force}`, entry]);
+}
+
+async function insert(entry: string, data: string, force = false): Promise<void> {
+  await gopass(["insert", `--force=${force}`, entry], data);
+}
+
+interface showAllResponse {
+  password: string;
+  attributes: string[];
+}
+async function showAll(entry: string): Promise<showAllResponse> {
```

Comment:
I’m not sure if we really need this function. We should either remove `showAll()` and replace it with calls to `password()` and `show()` or remove `password()` and rewrite `show()` to return password and details similar to this function. I'm not sure which way I'd prefer. What do you think?

Reply:
So I went with rewrite the old `show`. initially I added a new method to avoid large re-factors  but i didnt check how often that method is actually called - and its only once :-)

I wanted to avoid the multi-call option as there is no need to call the CLI twice imo

Reply:
That's a good point. I don't think it would have be a problem to call the CLI twice but your solution is way more clean, thank you :)